code/game/gamemodes/meteor/meteors.dm

```diff
@@ -319,31 +356,102 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 	if(!isspaceturf(T))
 		new /obj/effect/decal/cleanable/xenoblood(T)
 
-//Station buster Tunguska
-/obj/effect/meteor/tunguska
-	name = "tunguska meteor"
-	icon_state = "flaming"
-	desc = "Your life briefly passes before your eyes the moment you lay them on this monstrosity."
-	hits = 30
+//Blobby-Meaty Ore
+/obj/effect/meteor/meaty/mouldy
+	name = "mouldy ore"
+	desc = "Ewww."
+	color = "#654321"
+	meteorsound = 'sound/effects/splat.ogg'
+	meteordrop = list(/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/zombie, /obj/item/organ/tongue/zombie)
+	threat = 10
+	var/spawn_max = 4
+	var/blobbo_chance = 25
+
+/obj/effect/meteor/meaty/mouldy/meteor_effect()
+	..()
+	if(prob(blobbo_chance))
+		new /mob/living/simple_animal/hostile/blob/blobbernaut/independent(src.loc, src)
+	else
+		for(var/i in 1 to spawn_max)
+			var/mob/living/simple_animal/hostile/blob/blobspore/BS = new /mob/living/simple_animal/hostile/blob/blobspore(src.loc, src)
+			for(var/j = 1, j <= rand(1, 3), j++)
+				step(BS, pick(NORTH,SOUTH,EAST,WEST))
+
+//Bone-Meaty Ore
+/obj/effect/meteor/meaty/bone
+	name = "boney ore"
+	desc = "This thing sure drank its calcium."
+	icon = 'icons/obj/statuelarge.dmi'
+	icon_state = "skull"
+	meteordrop = list(/obj/item/stack/sheet/bone, /obj/item/organ/tongue/bone, /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/skeleton)
```

Comment:
we have those bone structures from the elephant graveyard ruin now; do any of them "fit" here?

Reply:
Not really, most of them are pretty big; I used one of them as the sprite for the meteor itself. It'd look odd if they popped out of the meteor.